contracts/Verifier.sol

```diff
@@ -404,4 +404,28 @@ library Verifier {
           }
           return verifyProof(a, b, c, inputValues, revealVerifyingKey());
```

Comment:
Just wondering where there isn't a huge diff here because we changed all the snark keys with our mimc adjustment

Reply:
i excluded it from the merge